opensrp-chw/src/main/java/org/smartregister/chw/dao/WashCheckDao.java

```diff
@@ -41,8 +41,34 @@ public static long getLastWashCheckDate(String familyBaseEntityID) {
         return res;
     }
 
+    public static void updateWashCheckVisitDetails(Long washDate, String baseEntityID,
```

Comment:
You can client process events without persisting them in the database. The Event will be updated in the visit table.
Check example of client processing without saving
`org.smartregister.chw.core.utils.NativeFormProcessor#clientProcessForm`